prov/verbs/src/verbs_eq.c

```diff
@@ -36,13 +36,13 @@
 #include "fi_verbs.h"
 
 struct fi_info *
-fi_ibv_get_verbs_info(struct fi_info *ilist, const char *domain_name)
+fi_ibv_get_verbs_info(const struct fi_info *ilist, const char *domain_name)
 {
-	struct fi_info *fi;
+	const struct fi_info *fi;
 
 	for (fi = ilist; fi; fi = fi->next) {
 		if (!strcmp(fi->domain_attr->name, domain_name))
-			return fi;
+		    return (void *)fi;
```

Comment:
Please change the function signature to add const, so we don't end up casting away the const here.

Reply:
Thanks. I missed to apply your comment regarding an incorrect casting of `fi_info` entries